package.json

```diff
@@ -26,29 +33,31 @@
     "url": "https://github.com/PRX/dovetail-counts-lambda/issues"
   },
   "homepage": "https://github.com/PRX/dovetail-counts-lambda#readme",
+  "devDependencies": {
+    "targz": "^1.0.1",
+    "@biomejs/biome": "2.2.4",
+    "aws-sdk-client-mock": "^4.1.0",
+    "aws-sdk-client-mock-jest": "^4.1.0",
+    "dotenv": "^17.2.2",
+    "jest": "^30.1.3",
+    "nock": "^14.0.10"
+  },
   "dependencies": {
+    "@aws-sdk/client-dynamodb": "^3.891.0",
+    "@aws-sdk/client-kinesis": "^3.891.0",
+    "@aws-sdk/client-sts": "^3.891.0",
     "ioredis": "^4.0.0",
-    "lambda-log": "^2.0.0"
-  },
-  "devDependencies": {
-    "aws-sdk": "^2.912.0",
-    "dotenv": "^10.0.0",
-    "jest": "^26.6.3",
-    "prettier": "^2.3.2"
+    "lambda-log": "^3.1.0"
   },
   "jest": {
-    "globalSetup": "<rootDir>/test-setup.js",
-    "modulePathIgnorePatterns": [
-      "<rootDir>/build/"
+    "setupFiles": ["dotenv/config"],
+    "restoreMocks": true,
+    "moduleNameMapper": {
+      "^aws-sdk-client-mock-jest$": "<rootDir>/node_modules/aws-sdk-client-mock-jest/dist/cjs/jest.js"
```

Comment:
I think I did this for some specific reason... stemming from [this thread](https://github.com/m-radzikowski/aws-sdk-client-mock/issues/242#issuecomment-2495205605).

But, doesn't seem to hurt anything, so why not.  You've also got some integration-test-y stuff in here, but also doesn't hurt anything.  (And we may want to add integrations to this repo eventually).